docs/_getting-started/index.md

```diff
@@ -4,197 +4,311 @@ permalink: docs/getting-started
 ```
+ 
+ > **Note:** http://127.0.0.1:8545 this is the default address
+ 
+ To check your account balance use the command below.
+
+ ```shell
+  curl -X POST http://127.0.0.1:8545 \
+    -H "Content-Type: application/json" \
+   --data '{"jsonrpc":"2.0", "method":"eth_getBalance", "params":["0xca57f3b40b42fcce3c37b8d18adbca5260ca72ec","latest"], "id":5}'
+   ```
+
+A successful call will return a response below:
 
-### Check account balance
+![Create new account command](../../static/images/rpcbalanceresult.png){:width="70%"}
 
+
+so Geth returns the value without invoking Clef. Note that the value returned is in hexadecimal and WEI. To get the ETH value, convert to decimal and divide by 10^18.
+
+**→ Check list of accounts**
+
+Run the command below to get all the accounts.
 ```shell
-curl -X POST http://<GETH_IP_ADDRESS>:8545 \
-    -H "Content-Type: application/json" \
-   --data '{"jsonrpc":"2.0", "method":"eth_getBalance", "params":["<ADDRESS_1>","latest"], "id":1}'
+curl -X POST http://127.0.0.1:8545 \
+    -H "Content-Type: application/json" \
+   --data '{"jsonrpc":"2.0", "method":"eth_accounts","params":[], "id":5}'
 ```
 
-Getting the balance of an account does not require a signed transaction, so Geth returns the value without invoking
-Clef. Note that the value returned is in hexadecimal and WEI. To get the ETH value, convert to decimal and divide by 10^18.
+Follow the same step as the IPC Check account balance
+
+A successful call will return a response below:
 
-### Send ETH to accounts
+![Create new account command](../../static/images/rpclistaccounts.png){:width="70%"}
 
-Send 0.01 ETH from the account that you added ETH to with the Görli faucet, to the second account you created:
 
+**→ Send ETH to accounts**
+
+**Syntax:**
 ```shell
-curl -X POST http://<GETH_IP_ADDRESS>:8545 \
-    -H "Content-Type: application/json" \
-   --data '{"jsonrpc":"2.0", "method":"eth_sendTransaction", "params":[{"from": "<ADDRESS_1>","to": "<ADDRESS_2>","value": "0x9184e72a"}], "id":1}'
+    curl -X POST http://<GETH_IP_ADDRESS>:8545 \
+        -H "Content-Type: application/json" \
+       --data '{"jsonrpc":"2.0", "method":"eth_sendTransaction", "params":[{"from": "<ADDRESS_1>","to": "<ADDRESS_2>","value": "0x9184e72a"}], "id":1}'
 ```
 
-This action does require signing, so Clef prompts you to approve it, and if you do,
-asks you for the password of the account from which you are sending the ETH. If the password is correct,
-Geth proceeds with the transaction.
+**step 1:** convert value from Eth to Wei decimal
+Use this link to do the conversation: [Eth converter](https://eth-converter.com/)
 
-To check, get the account balance of the second account:
+**step 2:** convert decimal to Hexadecimal
+Use this link to do the conversion: [Rapidtable](https://www.rapidtables.com/convert/number/decimal-to-hex.html) and add 0x at the begging of the number.
+Example:  from 2386F26FC10000 to 0x2386F26FC10000
```

Comment:
The 'steps' here feel a bit out of place. There is no need to give such a detailed description of converting a number to hexadecimal.

A simple example would be sufficient. For example, you could say

> Example: `0.0241 ether` is `24100000000000000 wei`, and would be encoded as the hex string `"0x559ed283164000"` in the JSON-RPC API.